client/my-sites/upgrades/checkout-thank-you/index.jsx

```diff
@@ -174,8 +174,23 @@ const CheckoutThankYou = React.createClass( {
 		return [ GenericDetails ];
 	},
 
+	shouldHideFeaturesHeading() {
```

Comment:
Since it's only called once with a `!` in front of it could we rename this function into `shouldDisplayFeaturesHeading()` and simplify the condition instead?